src/main/java/io/orten/nano/model/User.java

```diff
@@ -0,0 +1,105 @@
+package io.orten.nano.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Date;
+
+@Entity
+@Table(name = "user")
+public class User {
+
+    @Id
+    @Column(name = "userID")
+    private long userID;

```

Comment:
Does it work having these as private?
Might prove problematic when parsing to and from JSON

Reply:
Its working